packages/composites/bob/src/endpoint/format.ts

```diff
@@ -59,21 +61,24 @@ export const execute: ExecuteWithConfig<ExtendedConfig> = async (request, contex
   const blockNumber = validator.validated.data.blockNumber
 
   const block = await provider.getBlock(blockNumber)
-  const _execute: ExecuteWithConfig<Config> = JSONRPC.makeExecute()
+  const _execute = JSONRPC.makeExecute(config)
   const response = await _execute(
     {
       ...request,
-      data: { ...request.data, method: 'eth_getBlockByHash', params: [block.hash, false] },
+      data: {
+        ...(request.data as JSONRPC.types.request.TInputParameters),
+        method: 'eth_getBlockByHash',
+        params: [block.hash, 'false'],
```

Comment:
This shouldn't be a string

```suggestion
        params: [block.hash, false],
```